Sources/MongoSwift/Operations/CountOperation.swift

```diff
@@ -0,0 +1,75 @@
+import mongoc
+
+/// An operation corresponding to a "count" command on a collection.
+internal struct CountOperation<T: Codable>: Operation {
+    private let collection: MongoCollection<T>
+    private let filter: Document
```

Comment:
My one question on this PR is how much we care to duplicate documentation on what these properties are. They are already all documented on the `count` method... is it worth pasting that here too? (same q applies to distinct.)

Reply:
as long as it doesn't mess with your jazzy count, I don't see a compelling reason to duplicate options here

Reply:
it's all private/internal, so jazzy won't care.